### PR TITLE
feat: Show decoded values for known contract events

### LINKED ISSUE #501: Show decoded values for known contract events
Followup of #142. The same functionality should be implemented for Token events tab.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #501

## Demo
firefox_endA8euh5Z

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] extend decoded events panel to tokens event

### CODE REVIEW

File: src/components/ContractEventsTableCondensed.vue [hunk 1]
```diff
@@ -74,19 +74,19 @@
           v-else
           :class="[
             'contract-events-table-condensed__data',
-            {'contract-events-table-condensed__data--expanded': isOpened.includes(index)}]">
+            {'contract-events-table-condensed__data--expanded': isExpanded.includes(index)}]">
           <expand-button
-            :is-collapsed="!isOpened.includes(index)"
+            :is-expanded="isExpanded.includes(index)"
```
Comment (janmichek) [inline src/components/ContractEventsTableCondensed.vue:79]: refactor not related to feature itself. The naming was annoying me

here is the most visible. I renamed the props root name just to `expand*`
```
<expand-button
            :is-collapsed="!isOpened.includes(index)"
```

File: src/components/TokenEventsTable.vue [hunk 2]
```diff
@@ -9,48 +9,72 @@
...
-          Name
+          Date
           <hint-tooltip>
-            {{ tokensHints.eventName }}
+            {{ tokensHints.date }}
           </hint-tooltip>
         </th>
         <th>
-          Date
+          Name
```
Comment (janmichek) [inline src/components/TokenEventsTable.vue:18]: I switched the columns for the consistency with contract events table

**Review by michele-franchi — APPROVED** [review]
Good job 👏

**Review by lukeromanowicz — APPROVED** [review]
Well done, looks good :heavy_check_mark:

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Pull request merged**